### ISSUE TITLE
Origin in SCFInterpolatedPotential

### ISSUE THREAD

**Issue by jngaravitoc** [adrn/gala#333]
Should the `orgin` parameters in the `SCFInterpolatedPotential ` potential be equal to the `com_xj` parameter?

**Comment by adrn** [comment]
No! I should remove the origin parameter for this potential, but (as we discussed earlier :)) also see the bug in #339
